draft-hopwood-coinbase-balance.rst

```diff
@@ -0,0 +1,174 @@
+::
+
+  ZIP: XXX
+  Title: Blocks should balance exactly
+  Owners: Daira-Emma Hopwood <daira-emma@electriccoin.co>
+  Credits: Jack Grigg
+           Kris Nuttycombe
+  Status: Draft
+  Category: Consensus
+  Created: 2024-07-02
+  License: MIT
+  Discussions-To: <https://github.com/zcash/zips/issues/864>
+
```

Comment:
```suggestion
  Pull-Request: <https://github.com/zcash/zips/pull/865>

```

Reply:
Hmm, I don't think this header makes sense now, given that we've moved to "merge early, update draft across multiple PRs".